android/gradle.properties

```diff
@@ -0,0 +1,5 @@
+SmallcaseGateway_kotlinVersion=1.7.0
+SmallcaseGateway_minSdkVersion=21
+SmallcaseGateway_targetSdkVersion=31
```

Comment:
@LonelyCpp The SDK is yet to be tested for targetSDK 31, will be downgrading this to 30 to minimise impact.
Same for Kotlin version 1.7.0 -> 1.4.0

Reply:
Ok